packages/sdk/src/types/Exports.ts

```diff
@@ -1,32 +1,427 @@
-export interface ExportResource {
-	exported_object: 'voucher' | 'redemption' | 'publication' | 'customer'
-	parameters?: {
-		order?: string
-		fields?: string[]
-		filters?: {
-			junction?: 'AND' | 'OR'
-			[filter_condition: string]: any
-		}
-	}
+export interface ListExportQuery {
+	limit?: number
+	order?: 'created_at' | '-created_at' | 'status' | '-status'
+	page?: number
+}
+
+export interface ListExport {
+	object: 'list'
+	data_ref: 'exports'
+	exports: Partial<ExportObject>[]
+	total: number
 }
 
-export interface ExportsCreateResponse {
+export type ExportResource = CreateExport
```

Comment:
Interface vs Type - please look at origin comment about the `DistributionsPublicationsCreateParams`